### DISCUSSION TITLE
Reference中的note field不显示

### DISCUSSION THREAD

**Discussion by Biblehome** [sjtug/SJTUThesis#1058] [提问] [answered]
维护者你好，
我的编译环境：latexmk/texlive/centos7
我的reference中一些项目需要添加note，例如：

    @misc{LLP_signature, 
    author = {Heather Russell}, 
    title = {LLP Overview}, 
    url = {https://hrussell.web.cern.ch/hrussell/graphics.html}, 
    note = {This work is modified from the original copy and licensed under the Creative Commons Attribution 4.0 International License. To view a copy of this license, visit \url{http://creativecommons.org/licenses/by/4.0/}.}
    }

但是编译后的pdf中这些note不显示，如图所示：
![微信图片_20240822134835](https://github.com/user-attachments/assets/3d0ca5d4-53dd-4e15-8f2b-1f60550c70c9)
请问应当如何解决？
谢谢。

**Comment by LogCreative** [comment, marked as answer]
可以向下面的位置添加 `gbannote=true`，并将 `note` 改为 `annote` 来添加注释。

https://github.com/sjtug/SJTUThesis/blob/f5b9b29fffe0353fbdca220ce9cf47c287af4aeb/setup.tex#L112

例如：

```latex
\documentclass{sjtuthesis}
\begin{filecontents}{refs.bib}
@misc{LLP_signature, 
    author = {Heather Russell}, 
    title = {LLP Overview}, 
    url = {https://hrussell.web.cern.ch/hrussell/graphics.html}, 
    annote = {This work is modified from the original copy and licensed under the Creative Commons Attribution 4.0 International License. To view a copy of this license, visit \url{http://creativecommons.org/licenses/by/4.0/}.}
}
\end{filecontents}
\usepackage[style=gb7714-2015,gbannote=true]{biblatex}
\addbibresource{refs.bib}
\usepackage{hyperref}
\begin{document}
    \nocite{*}
    \printbibliography
\end{document}
```

> [`biblatex-gb7714-2015` 手册](https://mirrors.sjtug.sjtu.edu.cn/CTAN/macros/latex/contrib/biblatex-contrib/biblatex-gb7714-2015/biblatex-gb7714-2015.pdf)：
> gbannote = true，false. default is false
> 为控制是否在文献条目后面输出由annotation 或annote 域提供的注释信息而增加
> 的选项。
> - gbannote=false，即默认不输出。
> - gbannote=true，输出注释信息。
>
> ---
> note 域在本样式中也做特殊用途...

**Reply by Biblehome** [reply]
已解决，谢谢！
